services/workflows-service/src/app.module.ts

```diff
@@ -43,7 +44,38 @@ import { WebhooksModule } from '@/webhooks/webhooks.module';
 import { BusinessReportModule } from '@/business-report/business-report.module';
 import { ScheduleModule } from '@nestjs/schedule';
 import { CronModule } from '@/workflow/cron/cron.module';
+import { plainToInstance } from 'class-transformer';
+import z from 'zod';
 
+export const validate = (config: Record<string, unknown>) => {
+  //   const _salt = env.HASHING_KEY_SECRET_BASE64 ? Base64.decode(env.HASHING_KEY_SECRET_BASE64) : env.HASHING_KEY_SECRET;
+
+  // if (_salt === undefined || !_salt) {
+  //   throw new Error('Invalid salt value: HASHING_KEY_SECRET_BASE64');
+  // }
+
+  // hashKey('check default salt value', _salt)
+
+  const envServerSchema = z
+    .object(serverSchema)
+    .refine(data => data.HASHING_KEY_SECRET || data.HASHING_KEY_SECRET_BASE64, {
+      message: 'At least one of HASHING_KEY_SECRET or HASHING_KEY_SECRET_BASE64 should be present',
+      path: ['HASHING_KEY_SECRET', 'HASHING_KEY_SECRET_BASE64'],
+    });
+
+  const result = envServerSchema.safeParse(config);
+
+  if (!result.success) {
+    const errors = result.error.errors.map(zodIssue => ({
+      message: zodIssue.message,
+      path: zodIssue.path.join('.'), // Backwards compatibility - Legacy code message excepts array
+    }));
+
+    throw new Error(JSON.stringify(errors, null, 2));
```

Comment:
Can you share a screenshot of how the error looks like in the terminal?

Reply:
When Slat Key isnt exists in any form(regular or base64):
![Uploading image.png…]()


When Slat Key is base64 is present in wrong format:
<img width="620" alt="image" src="https://github.com/ballerine-io/ballerine/assets/6435752/4a2e857f-b762-4b50-8c79-a3a551de32ea">

Reply:
When Slat Key isnt exists in any form(regular or base64):
<img width="944" alt="image" src="https://github.com/ballerine-io/ballerine/assets/6435752/df1fd4c9-20bb-4cef-8d65-f99858af68f9">


When Slat Key is base64 is present in wrong format:
<img width="620" alt="image" src="https://github.com/ballerine-io/ballerine/assets/6435752/4a2e857f-b762-4b50-8c79-a3a551de32ea">